site/pages/review-disruption/[disruptionId].page.tsx

```diff
@@ -347,6 +342,28 @@ const ReviewDisruption = ({
                                                         ),
                                                     ],
                                                 },
+                                                {
+                                                    header: "Image",
+                                                    cells: [
+                                                        post.image ? (
+                                                            <Link
+                                                                className="govuk-link text-govBlue"
+                                                                key={post.image.key}
+                                                                href={post.image?.url ?? ""}
+                                                            >
+                                                                {post.image.originalFilename}
+                                                            </Link>
+                                                        ) : (
+                                                            "No image uploaded"
+                                                        ),
+                                                        createChangeLink(
+                                                            "hootsuite-profile",
+                                                            CREATE_SOCIAL_MEDIA_POST_PAGE_PATH,
+                                                            disruption,
+                                                            nextIndexSocialMedia,
```

Comment:
This should use the socialMediaIndex on the post item rather than `nextIndexSocialMedia`